### DISCUSSION TITLE
How to run `log` when the run is currently running

### DISCUSSION THREAD

**Discussion by legezam** [nextflow-io/nextflow#5010] [Q&A] [answered]
Hello,

I have a long running pipeline which I would like to gather status information from while it is running. I have submitted it with `-bg` parameter and `-name foo`

When it is running, if i run `nextflow log foo`, i get this error:
```
ERROR ~ Unable to acquire lock on session with ID 9a52e2a6-6883-4ab3-9d92-2d58aceaf45c

Common reasons for this error are:
 - You are trying to resume the execution of an already running pipeline
 - A previous execution was abruptly interrupted, leaving the session open

You can see which process is holding the lock file by using the following command:
 - lsof cwd/.nextflow/cache/9a52e2a6-6883-4ab3-9d92-2d58aceaf45c/db/LOCK
 ```

How shall i do this properly? After the entire run finishes, i am able to use `log` but i would need something even during the run.

Thank you!

**Comment by bentsherman** [comment, marked as answer]
It is not possible to use the `log` command for a run while it is still running, because the underlying cache db for that run can only be open by one reader at a time. You can use the console or log output instead
